java/src/jmri/jmrit/logix/SpeedUtil.java

```diff
@@ -471,11 +464,16 @@ private float getMomentumFactor(Element cv) {
         float num = 0;
         if (attr != null) {
             try {
-                 num = Integer.parseInt( attr.getValue());
-                // even with instant speed change, allow some time for new speed to be attained
-                // therefore add 1.  (.896 is NMRA spec)
-//                return (num + 1) * 896 / 28;     // milliseconds per step
-                num = (num + 1) * 896 * getThrottleSpeedStepIncrement();     // milliseconds per step
+                 /*  .896sec per  is NMRA spec for each CV value
```

Comment:
Should this be `.896sec per ??? is NMRA spec for each CV value` where ??? should be replaced with something?